Tools/PRWatcher.ps1

```diff
@@ -0,0 +1,193 @@
+				$invalidColor = "green"
+				$copyClip = $true
+			}
+			
+			if ($cleanOut -eq "Added") {
+				Write-Host -f $invalidColor "$timevar Error reading package identifier"
+				$noRecord = $true
+			}elseif ($wingetOutput -eq "No package found matching input criteria.") {
+				if ($noNew -eq $true) {
+					$noRecord = $true
+				}
+				Write-Host -f $invalidColor $timevar ($cleanOut) $wingetOutput
+			}elseif ($prVersion -eq "") {
+				$noRecord = $true
+				Write-Host -f $invalidColor "$timevar Error reading PR version"
+			}elseif ($wingetVersion -eq "Unknown") {
+				Write-Host -f $invalidColor "$timevar Error reading Winget version"
+			}elseif ($wingetVersion -eq "input") {
+				$noRecord = $true
+				Write-Host $wingetOutput
+			}elseif ($wingetVersion -eq $null) {
+				Write-Host $wingetOutput
+			}elseif ($wingetVersion -eq "add-watermark") {
+				$noRecord = $true
+				Write-Host -f $invalidColor "$timevar Error reading package identifier"
+			}elseif ($prVersion -gt $wingetVersion) {
+				Write-Host -f $validColor "$timevar $cleanOut prVersion $prVersion is greater than wingetVersion $wingetVersion"
+			}elseif ($prVersion -lt $wingetVersion) {
+				$outMsg = "$timevar $cleanOut prVersion $prVersion is less than wingetVersion $wingetVersion"
+				Write-Host -f $invalidColor $outMsg
+				if ($copyClip) {
+					$outMsg | clip
+				}
+			}elseif ($prVersion -eq $wingetVersion) {
+				Write-Host -f $cautionColor "$timevar $cleanOut prVersion $prVersion is equal to wingetVersion $wingetVersion"
+			}else {
+				$wingetOutput
+			};
+			$oldclip = $clip
+			$carryClip = $prVersion 
+			if ($noRecord -eq $false) {
+				if ($clip.length -le 128) {
+				$clip = $clip -join "" | where {$_ -match "[#][0-9]{5}"}
+				#Write-Host "Output $clip to $LogFile"
+				$clip | Out-File $LogFile -Append
+			} else {
+				Write-Host -f $cautionColor "$timevar Item length greater than 128 characters."
+			} 
+			}
+		};
+		sleep 1
+	}
+}
+
+<#Bug: Non-semantic version numbers get garbled.
+Example: https://github.com/microsoft/winget-pkgs/search?o=desc&q=chrisant996.Clink&s=committer-date&type=commits
+
+$carryClip sections are an unfinished attempt to address this bug.
+#>
+
+
+#Utility functions
+#Extract package name from clipboard contents
+function Get-CleanClip {
+	[CmdletBinding()]
+	param(
+		$out = (Get-Clipboard)
+	)
+	$out = $out -replace "_"," "
+	$out = $out -join "" #removes accidental line breaks
+	#$DebugPreference = 'Continue'
+	$i = 0
+	Write-Debug "$i time: $out";$i++
+	$out =  $out -replace "Add "
+	$out =  $out -replace "Automatic deletion of "
+	$out =  $out -replace "Automatic update of "
+	$out =  $out -replace "Remove "
+	$out =  $out -replace "Update "
+	Write-Debug "$i time: $out";$i++
+	if ($out.contains(": ")) {
+		$out =  ($out -split ": ")[1]
+	}
+	Write-Debug "$i time: $out";$i++
+
+	$out =  ($out -split " ")[0]
+	Write-Debug "$i time: $out";$i++
+	$out	
+}
+
+#Minimize output for automation
+function Search-Winget ($term) {
```

Comment:
```suggestion
function Search-WinGet ($term) {
```
nit: To match the capitalization used in README.md and PowerShell cmdlets
Would also need to update it everywhere in `PRWatcher.md`

Reply:
This has been fixed.

Reply:
This has been fixed.